recipes/espeak-ng/COPYING

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This is included in the upstream tarball, and probably isn't being used.

Reply:
I am still rather a newbie regarding contributing to `conda-forge`. What action should I take then?

Reply:
luckily that part's easy: just delete the vendored `COPYING` file, and it see if it builds.

I do see there are bunch of _additional_ `COPYING` files upstream for vendored things, these should be included as well, below in `about`:

```yaml
about:
  #... what's already there, except `license` and `license_file`
  license: GPL-3.0-only
  license_file:
  - COPYING
  - COPYING.APACHE
  - COPYING.BSD2
  - COPYING.UCD
```

Reply:
Thank you, will attend this shortly.

Reply:
@bollwyvl Done, I would really appreciate if you could give it another check

Reply:
licensing looks fine. 

It seems like upstream makes a serious effort to support non-Linux platforms, and we should _probably_ do the same.

I can't really provide much feedback on the rest, and especially if you can get it building on `win-64` and `osx-64`, you can request a review from `@conda-forge/help-python-c @conda-forge/help-c-cpp`